_targets.R

```diff
@@ -3,12 +3,14 @@ library(targets)
 tar_option_set(packages = c(
   'tidyverse',
   'ggdist',
-  'sf'
+  'sf',
+  'cowplot'
 ))
 
 source('2_process/src/temp_utils.R')
 source('2_process/src/prep_intervals.R')
 source('3_visualize/src/plot_gradient.R')
+source('3_visualize/src/plot_gradient_legend.R')
```

Comment:
Instead of creating a new, very similar function to make the legend, it might make more sense to adapt the previous one for both applications.